e2e/cephfs.go

```diff
@@ -312,6 +313,36 @@ var _ = Describe("cephfs", func() {
 					}
 				})
 			}
+			By("verify generic ephemeral volume support", func() {
+				// generic ephemeral volume support is beta since v1.21.
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 21) {
```

Comment:
it's skipping for 1.20 but as per https://github.com/ceph/ceph-csi/pull/2603#discussion_r739026205 it's agreed to run on  1.20 also?

Reply:
this was deliberate, considering this is beta and supported since 1.21, this should be good I believe.

Reply:
agreed, but in that case it is not needed to enable the featue gate in `minikube.sh`?

Reply:
it was added considering the script can be used independently for kube deployments, but may be here, we can skip that and only have e2e check. adjusted the PR..